distribution/bin/generate-binary-license.py

```diff
@@ -133,11 +133,11 @@ def print_license_name_underbar(license_name):
 
 def generate_license(apache_license_v2, license_yaml):
     print_log_to_stderr("=== Generating the contents of LICENSE.BINARY file ===\n")
-    
```

Comment:
nit these changes looks accidental.

Reply:
@cryptoe That's just my editor removing trailing whitespace.

Reply:
We generally try to avoid such changes.